src/project/server/routes/api.js

```diff
@@ -0,0 +1,114 @@
+// Define your endpoints here (this is your "controller file")
+const express = require('express');
+const ItemManager = require('../services/item-manager.js')
+
+const todoRouter = express.Router();
+const itemManager = new ItemManager();
+
+todoRouter.get('/', async(req, res) => {
```

Comment:
there shouldn't be logic in the controller, it is meant to just get what is needed aout of the requet and then use services that do the logic